### ISSUE TITLE
Environment variable not being set

### ISSUE THREAD

**Issue by tssva** [rupor-github/wsl-ssh-agent#9]
Running version 1.4 on 64-bit Windows 10 Pro Version 1909 OS Build 18363.535 and WSL version 1. The environment variable is not exported to WSL. I have tried using `-setenv` on its own and in combination with `-envname` to set a variable name. I have also tried both combinations with and without specifically specifying a socket path using `-socket`.

**Comment by rupor-github** [comment]
You need to start WSL session **after** wsl-ssh-agent-gui, there is no way to force the environment of already running process. If this does not help - add "--debug" to the command line and run [DebugView](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview) to see what is going on.  Share the log here - we may get lucky.

**Comment by tssva** [comment]
I figured out what the issue seems to be. During testing I tried two scenarios. In both I was launching wsl-ssh-agent gui from the command line of a powershell session. In one scenario I was launching WSL using the wsl command afterwards in the same powershell session. In the other scenario I was running the powershell session in the Microsoft Terminal preview and after launching wsl-ssh-agent-gui launched a new WSL tab in the app. In both case the environment variable was not passed. If I close Microsoft Terminal completely, restart it and then launch a WSL tab it correctly gets the environment variable. Also if I launch the WSL session from outside Microsoft Terminal or the powershell session WSL correctly gets the environment variable.
